infrastructure/cdn-in-a-box/traffic_ops/trafficops_adduser.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
```

Comment:
Is this file used? Looks like duplicate functionality of `traffic_ops/app/db/adduser.pl`

Reply:
I wanted to avoid having the Perl dependency to do this and have it done directly in the db container -- still evaluating best way to accomplish this..